### PR TITLE
two-bucket: implemented exercise

### LINKED ISSUE #734: two-bucket: implement exercise
There are exercises that are not yet implemented in Python track (the full list of them - https://github.com/exercism/python/issues/417#issuecomment-334841778).
You can help us change this situation by implementing this one. Ref #417

The guide ["Implement an exercise from specification"](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md) will help you succeed.

If you still have questions, I encourage you to ask them before starting working on the issue.

I also suggest claiming the issue by creating an empty PR as stated [here](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md#avoiding-duplicate-work)

Please note that there are some special ['closing issue' keywords](https://help.github.com/articles/closing-issues-using-keywords/) that you can put in description of your PR and Github will take care of closing the corresponding issue automatically after merge.
This simple thing helps us a lot, and makes our life a bit easier

### PR DESCRIPTION
**Pull request by parthsharma2**
Implemented the "two-bucket" exercise.
Resolves #734

### CODE REVIEW

**Comment by parthsharma2** [comment]
The Travis CI build failed, can anyone help me understand why this is failing?

File: config.json [hunk 1]
```diff
@@ -835,6 +835,16 @@
       ]
     },
     {
+      "uuid": "8c89f739-05fb-7b80-b5f9-6ad079c750ba8302be8",
+      "slug": "two-bucket",
+      "difficulty": 5,
+      "topics": [
+        "algorithms",
```
Comment (m-a-ge) [inline config.json]: We've switched to `snake_case` naming convention for topics to increase consistency among tracks, so could you please choose them from the following list - https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt

File: exercises/two-bucket/two_bucket.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def two_bucket(bucket_one, bucket_two, d, first):
```
Comment (m-a-ge) [inline exercises/two-bucket/two_bucket.py]: Is it possible to change `d` to something more meaningful? 

A good Python style guide from Google - https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

File: config.json (hunk 1 above)
Comment (parthsharma2) [reply config.json]: Sure 👍

**Comment by parthsharma2** [comment]
Made the changes

File: exercises/two-bucket/example.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  4.Repeat steps 1,2,3 till either the first or second bucket contains the
+    desired amount.
+'''
+
+
+def two_bucket(bucket_one_cap, bucket_two_cap, desired_liters, first):
+    if first == "one":
+        moves = 0
+        bucket_one, bucket_two = 0, 0
+        while(True):
```
Comment (m-a-ge) [inline exercises/two-bucket/example.py]: Honestly, I don't like these infinity loops. Perhaps we should revisit it later.

**Comment by m-a-ge** [comment]
@parthsharma2, merged, thanks.

It's a best practice to squash your commits https://github.com/exercism/docs/blob/master/contributing/git-basics.md#squashing. It isn't a big deal since I can do it myself, but it would be awesome to just click on 'Merge pull request' button 😃

Just for future reference a guide of how to write good commit messages - https://chris.beams.io/posts/git-commit/#seven-rules

**Comment by parthsharma2** [comment]
@m-a-ge thanks for the tips. I'll surely squash all my commits in any other future PRs 😃

**Pull request merged**